vike/node/cli/bin.ts

```diff
@@ -1,49 +1,21 @@
 import { cac } from 'cac'
-import { resolve } from 'path'
-import { runPrerenderFromCLI, runPrerender_forceExit } from '../prerender/runPrerender.js'
-import { projectInfo, assertUsage, assertWarning } from './utils.js'
-import pc from '@brillout/picocolors'
+import { projectInfo, assertUsage } from './utils.js'
+import { prerender } from './commands/prerender.js'
+import { eject } from './commands/eject.js'
+import { scaffold } from './commands/scaffold.js'
```

Comment:
It felt odd to have everything in bin.ts

Reply:
In principle, I agree, but let's discuss this kind of things later. This will actually heavily conflict with what I'm currently working on (the Vike CLI fully replacing Vite's CLI) so I'm inclined to think it's best we revert this refactor.